core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -3251,6 +3253,8 @@ object ZIO {
         )
     }
 
+  private[zio] def unitFn[A]: A => Unit = _UnitFn.asInstanceOf[A => Unit]
```

Comment:
This may be a stupid question on my part but do we need this? Isn't `Any => Unit` already a valid function for any place we want to use this? Or do we save time by doing the cast ourselves instead of the compiler having to realize that?

Reply:
Ah you're right, we don't need the helper function, just the `val`...